app/src/main/java/org/oppia/app/testing/InputInteractionViewTestActivity.kt

```diff
@@ -50,6 +54,7 @@ class InputInteractionViewTestActivity : AppCompatActivity(), StateKeyboardButto
 
   fun getPendingAnswerErrorOnSubmitClick(v: View) {
     fractionInteractionViewModel.checkPendingAnswerError(AnswerErrorCategory.SUBMIT_TIME)
+    numericInputViewModel.checkPendingAnswerError(AnswerErrorCategory.SUBMIT_TIME)
```

Comment:
Nit for consistency on order & elsewhere in the PR for interactions: please consistently make fractions before or after numeric input rather than switching them.

Reply:
@BenHenning We are not switching them we are calling the checkPendingAnswerError for the respective inputs from their viewModels... Could you please elaborate if you are not speak about the same.

Reply:
Done.

Reply:
@nikitamarysolomanpvt Suggest keep it other way around. fraction first and then numeric considering alphabetical order.

Reply:
We are following this order through out the app and test cases.